_authors/osci_2023/rahulsiddharth.markdown

```diff
@@ -0,0 +1,5 @@
+name: <Rahul Siddharth>
```

Comment:
Please add `---` at the top of the file and remove the angled brackets. The angled brackets signify a placeholder that should be replaced with your actual values.

Reply:
I did that, do you see the new pull request now?